flocker/provision/_install.py

```diff
@@ -128,15 +164,39 @@ def task_install_digitalocean_kernel():
     ])
 
 
-def task_upgrade_kernel_centos():
-    return sequence([
-        run_from_args([
-            "yum", "install", "-y", "kernel-devel", "kernel"]),
-        # For dkms and ... ?
-        run_from_args([
-            "yum", "install", "-y", "epel-release"]),
-        run_from_args(['sync']),
-    ])
+def task_install_requirements(distribution):
+    if distribution == 'ubuntu-14.04':
+        return sequence([
+            # Add ZFS repo for recent ZFS versions - XXX no minimum
+            # version documented
+            run_from_args([
+                "add-apt-repository", "-y", "ppa:zfs-native/stable"]),
+            # Add Docker repo for recent Docker versions - XXX no
+            # minimum version documented
+            run_from_args([
+                "add-apt-repository", "-y", "ppa:james-page/docker"]),
+            # Add ClusterHQ repo for installation of Flocker packages.
+            run_from_args([
+                "add-apt-repository", "-y",
+                "deb http://build.clusterhq.com/results/omnibus/master/ubuntu-14.04 /"]),  # noqa
+            # Update to read package info from new repos
+            run_from_args([
+                "apt-get", "update"]),
+            # Not clear that an upgrade is required at this point, so
```

Comment:
Remove this unless an upgrade causes some acceptance tests to pass on a supported platform which don't pass otherwise.